lib/db.py

```diff
@@ -49,7 +49,10 @@ class Club(Base):
     name = Column(String(250), nullable=False)
     school_id = Column(Integer, ForeignKey('school.id'))
     description = Column(String(1024), nullable=False)
-    img_type = Column(String(100), nullable=False)
+    img_type = Column(String(100), nullable=True)
+    category = Column(String(100), nullable=False)
+    location = Column(String(256), nullable=False)
+    usualTime = Column(String(256), nullable=False)
```

Comment:
try to sync up with @lindseyma with how to use the calendar functionality to make a google cal api call.... since the club gets associated with a calendar bc of the part of hte work that lindsey did, you'll get calendar functionality _almost_ for free

Reply:
the `location` and `usualTime` isn't meant to be tied to a specific event, but more of like general info to put on the club page (usual meeting location and usual meeting time)

when club admins finalize a meeting, they can make an announcement/create an event for their members to RSVP